PersonalTwitterFeed/src/personaltwitterfeed/PersonalTwitterFeed.java

```diff
@@ -52,6 +58,26 @@ public static void main(String[] args) {
         }
         
         System.out.println("Your twitter feed is full");
+        
+    }
+    
+    
+    
+    public static void main(String[] args) {
+        
+        Scanner keyboard = new Scanner(System.in);
+        
+        System.out.println("Welcome to your personal Twitter!");
+        System.out.println("What's your name, tweeter?");
+        
+        String tweeterName = keyboard.nextLine();
+        
+        System.out.println("Nice to meet you " + tweeterName + "!");
+        System.out.println("Enter your tweets and I will add them to your timeline!");
+      
+      newTweet(tweeterName);
```

Comment:
I like how you called the method, keeping the introduction lines while moving the loops to newTweet.